project/Dependencies.scala

```diff
@@ -3,7 +3,7 @@ import Keys._
 
 object Dependencies {
   val Versions = Seq(
-    crossScalaVersions := Seq("2.10.5", "2.11.6"),
+    crossScalaVersions := Seq("2.11.8", "2.11.0"),
```

Comment:
Not sure, I'll change the second version to 2.10.6.  I know that spark 2.x defaultly uses 2.11.8